README.md

```diff
@@ -183,10 +183,9 @@ identity provider, such as (2018/2) [solid.community](https://solid.community/),
 
 #### WebID_TLS
 
-A WebID profile from one of the Solid-compliant [identity providers](https://solid.github.io/solid-idps/), such as [databox.me](https://databox.me/),  
+WebID-TLS currently only works in Firefox at the moment (Oct 2018)
```

Comment:
@perploug Yeah I noticed that Chrome didn't give me the popup anymore lately. Would that still be possible behind a flag?

@melvincarvalho @timbl @kidehen You are client certificate users, what browser(s) work(s) for you?

Reply:
@RubenVerborgh , 

There are no browser specific issues that impede my use of WebID-TLS. It just works :) 

/cc @melvincarvalho @timbl

Reply:
@perploug But it didn't work for you in other browsers?

Reply:
I use TLS certs to log in to  with chrome, and this still works for me.

Reply:
I use WebID-TLS on chrome and firefox.  No issues here either.

Reply:
I don't use the WebId-TLS option as there is it apparently not supported on the solid.community pods (I seem to recall a comment that this option was being deprecated) 

The reason the firefox comment is in there is because it was there before, I just moved to the beginning of the sentence - but based on your comments it seems it can be removed all together - will update the PR

And to confirm I do get prompted for a certificates on Osx Chrome and firefox.

Reply:
WebID-TLS has always been the primary login system for solid.  It is not deprecated.  The plan is to support it on solid.community in future.  But code needs to be there first.

Reply:
databox is the provider that supports WebID-TLS, but it doesnt yet support WebID-OIDC

There is in theory a way where you wrap WebID-TLS inside WebID-OIDC, which also works today, or should work today.

However, by nature the TLS is more secure because it involves two parties, and OIDC is less secure because it requires a trusted third party.  On the other hand, the trusted third party has a good network effect today, due to the nature of large email and identity providers.

Over time, it would be good for both protocols to be supported.  That was the design aim, but we are not there yet.